src/components/FiatOnrampModal/index.tsx

```diff
@@ -79,6 +80,15 @@ export default function FiatOnrampModal() {
   const [error, setError] = useState<string | null>(null)
   const [loading, setLoading] = useState(false)
 
+  const { height: windowHeight } = useWindowSize()
+  const modalHeight = useMemo(() => {
+    if (!windowHeight) {
+      return 620 // use this as a fallback if we can't get the window height
+    }
+    const modalHeightPx = windowHeight - theme.navHeight
+    return Math.min(Math.round((modalHeightPx / windowHeight) * 100), 80)
+  }, [theme.navHeight, windowHeight])
```

Comment:
slightly confused by what's happening here - so do you want it to be 80% of the window height? why cant you just pass in `80vh` with css instead of doing math

Reply:
@tinaszheng this math was intended to calculate the height of the screen minus the navbar height. but after testing some more, i realized we were always falling back to 80vh because it was smaller. i think using a constant 80vh every time is honestly good enough for this fix, it works well consistently on all devices